web/docs/guides/auth/auth-apple.mdx

```diff
@@ -0,0 +1,218 @@
+---
+id: auth-apple
+title: "Apple Login"
+description: Setting up Apple Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description
+
+How to set up Apple Login ("Sign In With Apple")
+
+
+The setup process consists of six parts:
+
+* obtain an `App Id` with “Sign In with Apple” capabilities
+* obtain a `Services Id` - this will serve as the `client_id`
+* obtain a `secret key` that will be used to get our `client_secret`
+* generate the `client_secret` using the `secret key`
+* Add your `client id` and `client secret` keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
```

Comment:
Capitilize the first letter, and add full stops (same format as here: https://docs-git-docs-new-guides-supabase.vercel.app/docs#what-it-is)